### PR TITLE
chore(docutils): update consola to 3.2.3

### PR DESCRIPTION
**Pull request by eglitise**
This PR updates `consola` to v3. Most of the changes are copied from #18505, which this PR replaces. One difference: it seems like `consola` v3 `import` calls use ESM, so they have been changed to CJS `require()`.

I ran `./packages/docutils/bin/appium-docs.js` (after install) with various options and it seemed to work fine.

Also I'm not well versed in TS, so if something looks off or can be improved, please do point it out 🙏

### CODE REVIEW

File: packages/docutils/lib/cli/config.ts [hunk 1]
```diff
@@ -46,9 +45,9 @@ export async function findConfig(argv: string[] = hideBin(process.argv)) {
 
   // if --verbose is used, set the log level to debug.
   // otherwise use --log-level or the default.
-  let logLevel: LogLevel | keyof typeof LogLevelMap;
+  let logLevel: keyof typeof LogLevelMap;
   if (preArgs.verbose) {
-    logLevel = LogLevel.Debug;
+    logLevel = 'debug';
```
Comment (mykola-mokhnach) [inline packages/docutils/lib/cli/config.ts:50]: can we use LogLevels.debug ?

**Review by mykola-mokhnach — APPROVED** [review]

File: packages/docutils/lib/cli/config.ts (hunk 1 above)
Comment (eglitise) [reply packages/docutils/lib/cli/config.ts:50]: `logLevel` has the type `"silent" | "error" | "warn" | "info" | "debug"`, but `LogLevels.debug` is `number` . Similarly, `LogTypes.debug` is `Partial<LogObject>`.

File: packages/docutils/lib/constants.ts [hunk 2]
```diff
@@ -2,7 +2,8 @@
  * Constants used across various modules in this package
  * @module
  */
-import {LogLevel} from 'consola';
+// eslint-disable-next-line @typescript-eslint/no-var-requires -- Consola 3 import call is ESM
+const {LogLevels} = require('consola');
```
Comment (jlipps) [inline packages/docutils/lib/constants.ts:6]: I don't think we have done this elsewhere in Appium so far (mixed require and import). I'm not sure of potential consequences. At least so far we have just not upgraded any modules which are ESM only.

**Review by jlipps — APPROVED** [review]

File: packages/docutils/lib/constants.ts (hunk 2 above)
Comment (eglitise) [reply packages/docutils/lib/constants.ts:6]: I guess there's always the safe route of just blacklisting the package, but then I wonder why did the original PR have all of these changes added.

Comment (jlipps) [reply packages/docutils/lib/constants.ts:6]: hmm I'm not sure, do you know who was working on this earlier? was it @boneskull? maybe he can advise on the consequences of mixing import styles?

**Pull request merged**